### PR TITLE
 Add --latest and --all to podman mount/umount 

### PR DESCRIPTION
**Pull request by rhatdan**
I find these useful for playing around with containers.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
Sure, LGTM

File: cmd/podman/mount.go [hunk 1]
```diff
@@ -80,35 +85,38 @@ func mountCmd(c *cli.Context) error {
...
-	args := c.Args()
 	json := c.String("format") == of.JSONString
 	if !formats[c.String("format")] {
 		return errors.Errorf("%q is not a supported format", c.String("format"))
 	}
 
 	var lastError error
-	if len(args) > 0 {
-		for _, name := range args {
+	if len(mountContainers) > 0 {
```
Comment (TomSweeneyRedHat) [inline cmd/podman/mount.go]: Can the user still pass in a container name(s)/id(s)?  Is that handled without c.Args()?

Comment (rhatdan) [reply cmd/podman/mount.go]: Yes it should be handled in the helper function.

File: cmd/podman/mount.go [hunk 2]
```diff
@@ -80,35 +85,39 @@ func mountCmd(c *cli.Context) error {
 		}
 	}
 
+	if c.Bool("all") && c.Bool("latest") {
+		return errors.Errorf("--all and --latest cannot be used together")
+	}
+
```
Comment (adrianreber) [inline cmd/podman/mount.go]: I see you are using checkAllAndLatest(c) in `umount.go`, any reasons to not also use it here?

Comment (rhatdan) [reply cmd/podman/mount.go]: podman mount is allowed to have no containers.  And checkAllAndLatest returns an error if the user did not specify --all, or --latest, or containers.

podman mount 
will list all mounted containers.

File: cmd/podman/mount.go [hunk 3]
```diff
@@ -80,35 +85,39 @@ func mountCmd(c *cli.Context) error {
...
+	mountContainers, err := getAllOrLatestContainers(c, runtime, -1, "all")
+	if err != nil {
+		fmt.Println("DAN", err)
```
Comment (umohnani8) [inline cmd/podman/mount.go]: Remnant of your debugging?

**Comment by umohnani8** [comment]
One thing but LGTM

File: cmd/podman/mount.go (hunk 3 above)
Comment (rhatdan) [reply cmd/podman/mount.go]: oops

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
This is ready to merge.
@mheon @TomSweeneyRedHat @baude @giuseppe @vrothberg PTAL

**Pull request merged**